scripts/update_birdnet_snippets.sh

```diff
@@ -14,6 +14,7 @@ else
 fi
 my_dir=$HOME/BirdNET-Pi/scripts
 source "$my_dir/install_helpers.sh"
+source "$my_dir/install_service.sh"
```

Comment:
Wow indeed i blanked out on this one. Not sure why I thought it provisioned the function but you're totally right it would execute it which is not at all the intent! Especially to call a single function.

Regarding the logic, I was thinking that by looking at the number of occurence of #birdnet, we might delete all cron and restore them all which could be more efficient than by looking one by one using a tool like sed : indeed if the match is not found we would not know if the line was not existing at all, or if it was modified... That's why I was thinking about more a blank delete of cron and reinstall 

Speaking of birdnet.conf install, wouldn't it be much more efficient if instead of installing defaults through the update snippets, it would be handled by the get_config function? So that this function would get the parameter from birdnet.conf, or write the default value if it doesn't find it. This would avoid the complex updatesnippets script to update birdnet.conf, and would also take into account cases where use delete birdnet.conf values in between updates. Currently it fails silently by just not updating the value I think

Reply:
Of course it's your repo so if you prefer that I put a line-by-line check of the crons I'll do that I honestly don't mind ;)